rel-eng/build-packages-for-obs.sh

```diff
@@ -118,6 +118,11 @@ while read PKG_NAME PKG_VER PKG_DIR; do
     continue 2
   fi
 
+  # Adjust SPEC version to Uyuni (if needed)
+  if [ "$(grep -E '^web.version.uyuni\s*=\s*.+$' ../web/conf/rhn_web.conf)" != "" ]; then
```

Comment:
But when we add this to the config it is always present.
We submit from 1 source to 4.1 and Uyuni. I think we need a different indicator if we need add it to the version or not.
Or what do you plan?

Reply:
This is the plan:
- master will ALWAYS have `web.version.uyuni`, and will be used for the WebUI and whatever other parts `web.version` is being used now.
- When we want to submit milestones for SUSE Manager 4.1, `web.version.uyuni` is commented. After the submission, it's uncommented again (we did something like this for 4.0, at that time we changed the value of `web.version`.
- When we finally branch SUSE Manager 4.1 to its own internal branch, we remove `web.version.uyuni`

Reply:
This plan is not valid anymore. The latest changes are now adapted so `web.version.uyuni` can always be there, as a parameter is required at the scripts to submit for Uyuni. `spacewalk-schema-migration` was also updated to be able to handle it.